### PR TITLE
add support NetStandard2.0

### PR DESCRIPTION
**Pull request by genusP**

### CODE REVIEW

**Comment by genusP** [comment]
The title does not understand the purpose of PR147
My PR additionally contains:
 - Information about author, copyright and other info need for nuget pack
 - Update travis config

File: Asterisk.2013/Asterisk.NET/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AsterNET")]
-[assembly: AssemblyCopyright("Copyright © 2017")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-[assembly: Guid("abe98502-ea83-4b04-98c3-ffe3eabe06b0")]
-[assembly: AssemblyVersion("1.3.0.0")]
-[assembly: AssemblyFileVersion("1.3.0.0")]
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/Properties/AssemblyInfo.cs]: Why is the AssemblyInfo.cs file being deleted?
Is this information saved elsewhere now?

File: Asterisk.2013/Asterisk.NET/AsterNET.csproj [hunk 2]
```diff
@@ -1,42 +1,18 @@
...
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
-    <SccProjectName>SAK</SccProjectName>
-    <SccLocalPath>SAK</SccLocalPath>
-    <SccAuxPath>SAK</SccAuxPath>
-    <SccProvider>SAK</SccProvider>
+    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <Version>1.3.0</Version>
+    <Copyright>Copyright © 2017</Copyright>
+    <Authors>Ben Merrills</Authors>
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/AsterNET.csproj]: Might want to ask Ben if he wants his name on here like this.
Could also just link to https://github.com/AsterNET instead.

File: Asterisk.2013/Asterisk.NET/AsterNET.csproj [hunk 3]
```diff
@@ -1,42 +1,18 @@
...
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
-    <SccProjectName>SAK</SccProjectName>
-    <SccLocalPath>SAK</SccLocalPath>
-    <SccAuxPath>SAK</SccAuxPath>
-    <SccProvider>SAK</SccProvider>
+    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <Version>1.3.0</Version>
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/AsterNET.csproj]: Version number moved here? Will it still show correctly in the DLL file's details?

File: Asterisk.2013/Asterisk.NET/AsterNET.csproj (hunk 2 above)
Comment (genusP) [reply Asterisk.2013/Asterisk.NET/AsterNET.csproj]: This data from current nuget package https://www.nuget.org/packages/AsterNET/

File: Asterisk.2013/Asterisk.NET/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (genusP) [reply Asterisk.2013/Asterisk.NET/Properties/AssemblyInfo.cs]: moved to project file

File: Asterisk.2013/Asterisk.NET/AsterNET.csproj (hunk 3 above)
Comment (genusP) [reply Asterisk.2013/Asterisk.NET/AsterNET.csproj]: yes

**Comment by genusP** [comment]
Information from AssemblyInfo move to project file. 
Dotnet tool include command pack ( dotnet pack <project-file>) for generate NuGet package (also available VS context menu ). This is command get information from project file.

Allow move version information to separate file (version.props) and include to project file.

**Pull request merged**